src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -334,62 +332,44 @@ const ImageSelector = ({ control, field, item, selectImage }) => {
         control={control}
         errorMessage={errorMessage}
         hidden
+        validate
         name={name}
-        validate={true}
         value={value}
       />
       <RegularText smallest placeholder>
         {infoText}
       </RegularText>
 
       {value ? (
-        <WrapperRow center spaceBetween>
-          <Image source={{ uri: value }} style={styles.image} />
-
-          <TouchableOpacity
-            onPress={() => {
-              onChange('');
-              setErrorMessage('');
-            }}
-          >
-            <Icon.Trash color={colors.error} size={normalize(16)} />
-          </TouchableOpacity>
-        </WrapperRow>
+        <>
+          <WrapperRow center spaceBetween>
+            <Image source={{ uri: value }} style={styles.image} />
+
+            <TouchableOpacity
+              onPress={() => {
+                onChange('');
+                setErrorMessage('');
+              }}
+            >
+              <Icon.Trash color={colors.error} size={normalize(16)} />
+            </TouchableOpacity>
+          </WrapperRow>
+          <RegularText smallest>{imageInfoText}</RegularText>
```

Comment:
shouldn't this be rendered only if set? so wrapping it with `{!!imageInfoText && ...}`?

Reply:
4f4769453427750e74db1b209268861c890b1939